server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -138,6 +156,19 @@ def validate_nfs_mount(deployment, address, path)
           return
         end
 
+        # 36 is the expected UID and GID of the share
+        if validate_nfs_uid(36, deployment.id) == false
```

Comment:
Any particular reason to have methods here instead of just:

```ruby
if File.stat("/tmp/fusor-test-mount-#{deployment_id}").uid != 36
  # ...
end
```